### PR TITLE
Add Dockerfile-njs with helper scripts/add_js_module.sh.

### PR DESCRIPTION
**Pull request by kmashint**
This is related to the Adobe item below and a discussion with Jan Michael Ong on dropping the Dockerfile-alpine variant yet still allowing support for the ngx_http_js_module.  This creates a variant of the default/Ubuntu Dockerfile as `Dockerfile-njs` that includes the ngx_http_js_module to support the njs JS-based extension language to nginx.

ETHOS-36876 BBC: Update behance/docker-nginx to use Ubuntu 22.04, deprecate Alpine, update nginx package repo

### CODE REVIEW

File: Dockerfile-njs [hunk 1]
```diff
@@ -0,0 +1,63 @@
+FROM behance/docker-base:4.0-ubuntu-20.04
```
Comment (adobejmong) [inline Dockerfile-njs:1]: @kmashint : Please consider updating to docker-base:5.0 if possible especially if this is going to be a brand new image anyway

File: README.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ Provides base OS, patches and stable nginx for quick and easy spinup.
 - Ubuntu used by default
 - Alpine builds available tagged as `-alpine`
 - Centos builds available tagged as `-centos`
+- Ubuntu with ngx_http_js_module tagged as `-njs`
```
Comment (adobejmong) [inline README.md:10]: @kmashint : I noted this in the JIRA as well but the name needs to change to be more consistent with the rest. I will get back to you tomorrow.

File: Dockerfile-njs (hunk 1 above)
Comment (kmashint) [reply Dockerfile-njs:1]: You also mentioned Ubuntu 22.04, so should I align on this?
`FROM behance/docker-base:5.0-ubuntu-22.04`

**Comment by kmashint** [comment]
Withdrawn since a different direction will be taken, to use nginx.org as the package repository for nginx and it would be up to other projects to add the ngx_http_js_module, but at least it would be more directly possible.

**Pull request closed without merging**